it/uk/gov/hmrc/minorentityidentificationfrontend/assets/TestConstants.scala

```diff
@@ -374,7 +368,22 @@ object TestConstants {
     "regime" -> regime
   )
 
-  def testRegistrationJourneyDataPart(value: String) = Json.obj("registration" ->  Json.obj("registrationStatus" -> value))
+  val testRegistrationNotCalledJson: JsObject = Json.obj("registrationStatus" -> "REGISTRATION_NOT_CALLED")
+
+  def testRegistrationJourneyDataPart(value: String): JsObject = registrationJson(content = Json.obj("registrationStatus" -> value))
+
+  def testBackendFailedRegistrationJson(failures: JsArray): JsObject = registrationJson(content = Json.obj(
+    "registrationStatus" -> "REGISTRATION_FAILED",
+    "failures" -> failures
+  ))
+
+  def testBackEndRegisteredJson(safeId: String): JsObject = registrationJson(content = testSuccessfulRegistrationJson(safeId))
+
+  def testSuccessfulRegistrationJson(safeId: String): JsObject = Json.obj(
+    "registrationStatus" -> "REGISTERED",
+    "registeredBusinessPartnerId" -> safeId)
```

Comment:
Can't we use below val declared in RegistrationStatus model. What's your thought please?

val registrationStatusKey = "registrationStatus"
  val registeredBusinessPartnerIdKey = "registeredBusinessPartnerId"
  val RegisteredKey = "REGISTERED"
  val RegistrationFailedKey = "REGISTRATION_FAILED"
  val RegistrationNotCalledKey = "REGISTRATION_NOT_CALLED"
  val registrationFailuresKey = "failures"